docs/Style-Guide.md

```diff
@@ -102,16 +102,16 @@ When writing E2E tests, follow [Cypress Best Practices](https://docs.cypress.io/
 
 ### Feedback Messages
 
-When the user performs an action (ex: submiting a form), it's expected to show them some type of feedback, on success or failure. There are 3 types of messages we want to display to the user:
+When the user performs an action (ex: submitting a form), it's expected to show them some type of feedback, on success or failure. There are three types of messages we want to display to the user:
 
 #### Type 1 - while filling a form
 
 These messages (mostly errors) are displayed whenever the user makes a mistake when filling out a form input. Examples:
 
 > A {fieldname} is required. —— Ex: A password is required.  
-> A {fieldname} cannot {action}. —— Ex: A username cannot contain spaces.  
+> A {fieldname} cannot {action}. —— Ex: An username cannot contain spaces.  
```

Comment:
Was correct as it was before

Reply:
After some research, I have to agree with you on this point. So today I learned something new about English elision rules